inlong-sort/sort-connectors/kafka/src/main/java/org/apache/inlong/sort/kafka/FlinkKafkaProducer.java

```diff
@@ -859,10 +861,14 @@ public void open(Configuration configuration) throws Exception {
                         @Override
                         public void onCompletion(RecordMetadata metadata, Exception e) {
                             if (e != null) {
-                                sendDirtyMetrics(rowSize, dataSize);
+                                sinkMetricData.sendDirtyMetrics(metadata.topic(), 1L, dataSize);
```

Comment:
It is inappropriate to determine whether it is dirty data here, because there may be network abnormalities, and it is recommended to be consistent with the judgment logic of dirty data archiving for dirty data